src/core/QCheck.mli

```diff
@@ -190,16 +190,33 @@ module Gen : sig
   val shuffle_l : 'a list -> 'a list t
   (** Creates a generator of shuffled lists. *)
 
-  val unit: unit t (** The unit generator. *)
+  val unit : unit t (** The unit generator. *)
 
-  val bool: bool t (** The boolean generator. *)
+  val bool : bool t (** The boolean generator. *)
 
-  val float: float t   (** Generates floating point numbers. *)
+  val float : float t   (** Generates floating point numbers. *)
 
   val pfloat : float t (** Generates positive floating point numbers (0. included). *)
 
   val nfloat : float t (** Generates negative floating point numbers. (-0. included) *)
 
+  val float_bound_inclusive : float -> float t
```

Comment:
need some `@since NEXT_RELEASE` annotations.

Reply:
Should I put `@since 0.10` or literally `@since NEXT_RELEASE`?

Reply:
literally `@since NEXT_RELEASE`, there is a makefile target to update that if you're curious :)